src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -197,6 +199,7 @@
 	private final ITileDefinition drive;
 	private final ITileDefinition chest;
 	private final ITileDefinition iface;
+	private final ITileDefinition fluidIface;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "fluidIface" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)